packages/odyssey-react/src/components/Tabs/Tabs.module.scss

```
@@ -12,26 +12,28 @@
 
 .tablist {
   position: relative;
-  border-block-end: 1px solid $border-color-display;
+  border-block-end-width: var(--BorderBlockEndWidthTabList);
+  border-block-end-style: var(--BorderBlockEndStyleTabList);
+  border-block-end-color: var(--BorderBlockEndColorTabList);
```

color related styles always start with the `Color` prefix:

```suggestion
  border-block-end-color: var(--ColorBorderBlockEndTabList);
```